File: src/main/java/kr/mjc/rachel/basics/jdbc/post/Post.java
package kr.mjc.rachel.basics.jdbc.post;

import lombok.Data;

@Data
public class Post {
  int postId;
  String title;
  String content;
  int userId;
  String name;
  String cdate;
  String udate;

  @Override
  public String toString() {
    return String.format(
        "\nPost{postId=%d, title=%s, content=%s, userId=%d, name=%s, cdate=%s, udate=%s}",
        postId, title, content, userId, name, cdate, udate);
  }
}

File: src/main/kotlin/kr/mjc/jacob/jdbc/post/Post.kt
package kr.mjc.jacob.jdbc.post

import kr.mjc.jacob.formatted
import java.time.LocalDateTime

class Post(var id: Int = 0, var title: String = "", var content: String = "",
           var userId: Int = 0, var firstName: String = "",
           var pubDate: LocalDateTime = LocalDateTime.now(),
           var lastModified: LocalDateTime = pubDate) {

  override fun toString(): String {
    return "Post(id=$id, title='$title', content='$content', userId=$userId, firstName='$firstName', pubDate=${pubDate.formatted}, lastModified=${lastModified.formatted})"
  }
}